.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+    browser: true,
+    es2021: true,
+  },
+  plugins: ['import', '@typescript-eslint', 'react', 'react-hooks', 'jsx-a11y'],
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:jsx-a11y/recommended',
+    'prettier',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: 'module',
+    tsconfigRootDir: __dirname,
+    project: [
+      'tsconfig.eslint.json',
+      './common/tsconfig.json',
+      './web/tsconfig.json',
+      './server/tsconfig.json',
+    ],
+  },
+  rules: {
+    // turn on errors for missing imports
+    'import/no-unresolved': 'error',
+
+    // warn us if we're using @ts-ignore etc..
+    '@typescript-eslint/ban-ts-comment': 'warn',
+  },
+  settings: {
+    react: {
+      version: 'detect',
```

Comment:
nit: getting a warning:  `Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.`

Could just use "latest" instead